crates/oxc_ast_macros/src/lib.rs

```diff
@@ -1,5 +1,14 @@
 use proc_macro::TokenStream;
-use std::str::FromStr;
+use proc_macro2::TokenStream as TokenStream2;
+use quote::quote;
+
+fn enum_repr(enum_: &syn::ItemEnum) -> TokenStream2 {

```

Comment:
Please add doc comment on this function clarifying when it's `#[repr(C)]` and when `#[repr(C, u8)]`.